include/seqcmd.h

```diff
@@ -0,0 +1,652 @@
+#ifndef SEQCMD_H
+#define SEQCMD_H
+
+// ==== Primary commands ====
+
+#define SEQ_CMD_MASK 0xF0000000
+
+typedef enum {
+    /* 0x0 */ SEQ_CMD_PLAY,
+    /* 0x1 */ SEQ_CMD_STOP,
+    /* 0x2 */ SEQ_CMD_QUEUE,
+    /* 0x3 */ SEQ_CMD_UNQUEUE,
+    /* 0x4 */ SEQ_CMD_SET_PLAYER_VOLUME,
+    /* 0x5 */ SEQ_CMD_SET_PLAYER_FREQ,
+    /* 0x6 */ SEQ_CMD_SET_CHANNEL_VOLUME,
+    /* 0x7 */ SEQ_CMD_SET_PLAYER_IO,
+    /* 0x8 */ SEQ_CMD_SET_CHANNEL_IO,
+    /* 0x9 */ SEQ_CMD_DISABLE_CHANNEL_IO,
+    /* 0xA */ SEQ_CMD_DISABLE_CHANNELS,
+    /* 0xB */ SEQ_CMD_TEMPO_CMD,
+    /* 0xC */ SEQ_CMD_SETUP_CMD,
+    /* 0xD */ SEQ_CMD_SET_CHANNEL_FREQ,
+    /* 0xE */ SEQ_CMD_GLOBAL_CMD,
+    /* 0xF */ SEQ_CMD_RESET_HEAP
+} SeqCmdType;
+
+// ==== Secondary commands ====
+
+// Subset of SEQ_CMD_TEMPO_CMD
+typedef enum {
+    /* 0x0 */ SEQ_SUB_CMD_TEMPO_SET,
+    /* 0x1 */ SEQ_SUB_CMD_TEMPO_SPEED_UP,
+    /* 0x2 */ SEQ_SUB_CMD_TEMPO_SLOW_DOWN,
+    /* 0x3 */ SEQ_SUB_CMD_TEMPO_SCALE,
+    /* 0x4 */ SEQ_SUB_CMD_TEMPO_RESET
+} TempoCmdType;
+
+// Subset of SEQ_CMD_SETUP_CMD
+typedef enum {
+    /* 0x0 */ SEQ_SUB_CMD_SETUP_RESTORE_VOLUME,
+    /* 0x1 */ SEQ_SUB_CMD_SETUP_SEQ_UNQUEUE,
+    /* 0x2 */ SEQ_SUB_CMD_SETUP_RESTART_SEQ,
+    /* 0x3 */ SEQ_SUB_CMD_SETUP_TEMPO_SCALE,
+    /* 0x4 */ SEQ_SUB_CMD_SETUP_TEMPO_RESET,
+    /* 0x5 */ SEQ_SUB_CMD_SETUP_PLAY_SEQ,
+    /* 0x6 */ SEQ_SUB_CMD_SETUP_SET_FADE_TIMER,
+    /* 0x7 */ SEQ_SUB_CMD_SETUP_RESTORE_VOLUME_IF_QUEUED,
+    /* 0x8 */ SEQ_SUB_CMD_SETUP_RESTORE_VOLUME_WITH_SCALE,
+    /* 0x9 */ SEQ_SUB_CMD_SETUP_DISABLE_CHANNELS,
+    /* 0xA */ SEQ_SUB_CMD_SETUP_SET_PLAYER_FREQ,
+    /* 0xE */ SEQ_SUB_CMD_SETUP_POP_CACHE = 0xE,
+    /* 0xF */ SEQ_SUB_CMD_SETUP_RESET_SETUP_CMDS
+} SetupCmdType;
+
+// Subset of SEQ_CMD_GLOBAL_CMD
+typedef enum {
+    /* 0x0 */ SEQ_SUB_CMD_GLOBAL_SET_SOUND_MODE,
+    /* 0x1 */ SEQ_SUB_CMD_GLOBAL_DISABLE_NEW_SEQUENCES
+} SubCmdType;
+
+// ==== Audio Sequence Primary Commands ====
+
+/**
+ * ARGS
+ *   playerIndex (p), fadeInTimer (t), seqArgs (a), seqId (s)
+ *
+ * FORMAT
+ *   Capital U is unused
+ *   0pttaass
+ *
+ * DESCRIPTION
+ *   Request a sequence (seqId) to be played on the specified player (playerIndex).
+ *   The sequence will gradually fade in over the course of (fadeInTimer).
+ *   How (fadeInTimer) is interpreted depends on (seqArgs)
+ *
```

Comment:
in all of these descriptions it would be preferable if the variable names are referenced with backticks instead: `var`
doxygen will recognize that, and it better conveys that its a variable name imo

Reply:
Would you suggestion backticks with or without the brackets?

Reply:
for something like "Request a sequence (`seqId`)", it makes sense to keep it since you already referred to it as "sequence" and are just clarifying the variable. 
but at the end "How (fadeInTimer) is interpreted depends on (seqArgs)"
you are referring to them directly, so no parenthesis makes more sense:
"How `fadeInTimer` is interpreted depends on `seqArgs`"